src/tracing/trace_event.h

```diff
@@ -0,0 +1,1710 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SRC_TRACING_TRACE_EVENT_H_
+#define SRC_TRACING_TRACE_EVENT_H_
+
+#include <stddef.h>
+
+#include "v8-platform.h"
```

Comment:
The platform is set here: https://github.com/matthewloring/node/blob/51842590b093297481e995d5be23294e238d18dc/src/node.cc#L207 and used by the macros:
https://github.com/nodejs/node/blob/51842590b093297481e995d5be23294e238d18dc/src/tracing/trace_event.h#L1217
https://github.com/nodejs/node/blob/51842590b093297481e995d5be23294e238d18dc/src/tracing/trace_event.h#L1239
https://github.com/nodejs/node/blob/51842590b093297481e995d5be23294e238d18dc/src/tracing/trace_event.h#L1247